graphql_compiler/cost_estimation/filter_selectivity_utils.py

```diff
@@ -1,8 +1,12 @@
 # Copyright 2019-present Kensho Technologies, LLC.
+from __future__ import division
```

Comment:
This is probably not necessary, I think all our divisions explicitly cast to float prior to dividing.

Reply:
It's the recommended py2 compatible way of doing it in `six`, and our linters complain about it too. Not sure if it's necessary either.

Reply:
Hmm okay. I thought that bug in `pylint` got fixed, but maybe we just haven't updated to the new version or something.